roachtest: add operation to query crdb_internal.probe_ranges

noahstho opened the pull request:
Since SRE uses crdb_internal.probe_ranges to test for prod cluster health, we would like to add this as a roach operation to make the DRT cluster as realistic as possible, and test for potential issues with crdb_internal.probe_ranges.

Fixes: #102034
Release note: None
Epic: None

noahstho committed:
roachtest: add operation to query crdb_internal.probe_ranges

Since SRE uses crdb_internal.probe_ranges to test for prod
cluster health, we would like to add this as a roach operation
to make the DRT cluster as realistic as possible, and test for
potential issues with crdb_internal.probe_ranges.

Fixes: #102034
Release note: None
Epic: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/143902)
<!-- Reviewable:end -->

noahstho committed:
roachtest: add operation to query crdb_internal.probe_ranges

Since SRE uses crdb_internal.probe_ranges to test for prod
cluster health, we would like to add this as a roach operation
to make the DRT cluster as realistic as possible, and test for
availability of crdb_internal.probe_ranges.

Fixes: #102034
Release note: None
Epic: None
